newsletter-november-2019.rst

```diff
@@ -0,0 +1,136 @@
+
+Looking forward to next year, the site for `Portland 2020 </conf/portland/2019/>`__ is live, as is the `Call For Proposals </conf/portland/2020/cfp>`__. We'd love to hear all your awesome talk ideas, so get working on them! The CFP closes on 10th January, 2020.
+
+Finally, we're changing policy for how we change policies, by introducing `Write the Docs Enhancement Proposals </blog/introducing-weps/>`__. The way we run the community has grown organically with the community itself, from seven whole years ago when it was just one conference . Our processes are only loosely defined - but that makes it harder for new people to understand the community and join the leadership. So we're trying WEPs, an idea taken from the open-source ecosystem. If you're interested, take a look at the `blog post </blog/introducing-weps/>`__, which explains it all much better.
+
+Onto this month's articles!
+
+--------------------------------------
+The strengths of different backgrounds
+--------------------------------------
+
+One of our most wide-ranging discussions this month put the breadth of documentarians' educational and work backgrounds on full display. It made pretty clear that technical documentation draws folks with an incredible variety of education and experience. Here's just a sample of some of the different backgrounds, and the strengths people feel they gained through them:
+
+* Culinary arts and restaurants: experience with managing stress, coping in sink-or-swim situations, and working with diverse groups of people
+* Customer support and tech support: lots of practice at explaining things; appreciation for the value of writing things down
+* Education: well prepared to develop training curriculum and knowledge base content
+* History, philosophy, religion, and ethics: strong reading, writing, and arguing skills, and especially the ability to write *quickly*
+* Library and information science: familiar with information architecture, tools, and technologies
+* Sociology and anthropology: keen human-centric insight and user-focused perspective
+* Technical writing certification or degree: allowed to grow into the role; driven to constantly improve
+
+We could fill an entire newsletter describing the paths people took to their roles in documentation. But no matter how our documentarians all found their way here, we're glad they did ❤️
+
+---------------------------------------------------
+Docs and design: When docs can't fix all the things
+---------------------------------------------------
+
+Much as we'd like to wish otherwise, we don't usually document perfectly usable software. (Is there such a thing?) And sometimes, they have major usability problems. A plaintive question about how to document a poorly designed product generated a range of responses - from how to provide feedback about the design and try to improve it, to how to hide or make less prominent less usable portions of the product.
```

Comment:
Not super keen on "like to wish". 

```suggestion
Much as we'd like to think otherwise, we don't usually document perfectly usable software. (Is there such a thing?) And sometimes, they have major usability problems. A plaintive question about how to document a poorly designed product generated a range of responses - from how to provide feedback about the design and try to improve it, to how to hide or make less prominent less usable portions of the product.
```